### PR TITLE
Feature navigation

### PR DESCRIPTION
**Pull request by zieeco**
# awesom book: Add navigation
**In this milestone, we completed the following**

- Added navigation to the main page which shows only the section a user clicks
- Ensured there were no linters error

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ This is the first project in the microverse javascript module.
 </p>
 
 This project is from the Microverse curriculum. It's main objective is to help student understand how to work with classes in javascript.
-Next we styled the project to match the framework design that was given using css.
+Next we styled the project to match the framework design that was given using css and finally we added the header which has th navigation to the sections in the page.
 
 ## Built With
```
Comment (Temmarie) [inline README.md:12]: - [ OPTIONAL] Good job here, your readme is descriptive but it needs a bit more work on the setup instructions. 
Kindly add more information describing the importance of your project, what it does and how to run it. This way, non-technical users can understand how to run it.  Currently, the readme doesn't show how to run the project after cloning it.

File: main.js [hunk 2]
```diff
@@ -96,5 +96,65 @@ document.addEventListener('DOMContentLoaded', () => {
...
+//   // document.getElementById('contact-link').classList.add('active')
+// })
+
+// const listLink = document.getElementById('list-link')
+// addNewLink.addEventListener('click', () => {
+//   document.getElementById('awesome-books').classList.remove('hide')
+//   document.getElementById('add-book').classList.add('hide')
+//   document.getElementById('contact').classList.add('hide')
+//   // document.getElementById('list-link').classList.add('active')
+// })
```
Comment (Temmarie) [inline main.js:160]: - [ ] Kindly remove all commented out codes as they make your project unreadable and unprofessional. If you need to check a code from previous commits, you can always use gitlog.

File: main.js [hunk 3]
```diff
@@ -96,5 +96,65 @@ document.addEventListener('DOMContentLoaded', () => {
...
+const contactSect = document.getElementById('contact-link');
+
+seeListOfBooks.addEventListener('click', () => {
+  document.getElementById('awesome-books').classList.remove('hide');
+  document.getElementById('add-book').classList.add('hide');
+  document.getElementById('contact').classList.add('hide');
+  seeListOfBooks.classList.add('active');
+  addNewBook.classList.remove('active');
+  contactSect.classList.remove('active');
+});
```
Comment (Temmarie) [inline main.js:118]: - [ ] Good job implementing the Date feature it works perfectly well. However, as per the instructions for this project, you are to use the `Luxom` library to implement this feature. 

image

**Review by Temmarie — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED! :recycle:

### Hi @zieeco    ! :wave: :wave:

#### You have done really well in implementing the requirements.
- You used proper GitHub Flow ✔️ 
- Your linter checks are all passing ✔️ 
- Your PR message is descriptive ✔️ 
- Your project matches the wireframe ✔️ 

 However, there are changes required for you to make in order for this project to be approved. 
- - -

### **Recommended** ♻️

_See the inline comments below for more required changes_


### Optional Changes 👌
Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better.
- N/A

_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

**_"Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."_**
- - - 
**_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._**
__Kindly make the necessary changes and submit another code review request.__

File: main.js (hunk 3 above)
Comment (rloterh) [reply main.js]: This issue is well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Isaac, 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer(s). There is nothing else to say other than... it's time to merge it 🍾:shipit:.
Congratulations! 🎉🥂

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**